src/language/sv.json

```diff
@@ -88,24 +90,24 @@
     "laatijat": {
       "table": {
         "laatija": "Upprättare",
-        "liitetty": "Liitetty SV?",
-        "hyvaksyja": "Hyväksyjä SV?",
-        "delete": "Poista SV?",
-        "proposal": "Odottaa hyväksymistä SV?"
+        "liitetty": "Kopplad",
+        "hyvaksyja": "Godkännare",
+        "delete": "Radera",
+        "proposal": "Inväntar godkännande"
       },
       "accept": {
-        "confirm-message": "Oletko varma että haluat liittää laatijan yritykseen. SV?",
-        "confirm-button": "Hyväksy SV?",
-        "link": "Hyväksy liittäminen SV?",
-        "success": "Laatijan hyväksyminen yritykseen onnistui SV?",
-        "error": "Laatijan hyväksyminen yritykseen epäonnistui SV?"
+        "confirm-message": "Är du säker på att du vill koppla upprättaren till företaget?  ",
```

Comment:
Aika paljon näitä missä on loppuun tullut whitespacea. Suomenkielisissä vastineissa tätä ainakaan enimmäkseen ei ole.

Reply:
Joo, käännöstentekijät eivät ole nähtävästi trimmannut näitä. Mergescriptin tarve kasvaa suuremmaksi.